css-text/Overview.bs

```diff
@@ -2226,50 +1856,17 @@ Cursive Scripts</h4>
   <h3 id="text-indent-property">
     First Line Indentation: the 'text-indent' property</h3>
 
-    <table class="propdef">
-        <tr>
-          <th>Name:</th>
-          <td><dfn>text-indent</dfn></td>
-        </tr>
-        <tr>
-          <th><a href="#values">Value</a>:
-          <td>[ <span class="value-inst-length"><a href="https://www.w3.org/TR/CSS21/syndata.html#value-def-length">&lt;length&gt;</a></span>
-              | <span class="value-inst-percentage"><a href="https://www.w3.org/TR/CSS21/syndata.html#value-def-percentage">&lt;percentage&gt;</a></span>
-              ] && hanging? && each-line?
-          </td>
-        </tr>
-        <tr>
-          <th>Initial:</th>
-          <td>0</td>
-        </tr>
-        <tr>
-          <th>Applies to:</th>
-          <td>block containers</td>
-        </tr>
-        <tr>
-          <th>Inherited:</th>
-          <td>yes</td>
-        </tr>
-        <tr>
-          <th>Percentages:</th>
-          <td>refers to width of containing block</td>
-        </tr>
-        <tr>
-          <th>Media:</th>
-          <td>visual</td>
-        </tr>
-        <tr>
-          <th>Computed&#160;value:</th>
-          <td>the percentage as specified or the absolute length,
-            plus any keywords as specified
-        </tr>
-        <tr>
-          <th>Animatable:
-          <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-lpcalc">length, percentage, or calc</a>, but only if keywords match
-        <tr>
-          <th>Canonical order:
-          <td><abbr title="follows order of property value definition">per grammar</abbr>
-    </table>
+    <pre class="propdef">
+    Name: text-indent
+    Value: [ <<length>> | <<percentage>> ] && hanging? && each-line?
```

Comment:
Done in https://github.com/w3c/csswg-drafts/pull/164/commits/9bc5074825ff8a68ce6e989af4196b0c28de3b70.

Sebastian